newswires/client/src/WireDetail.tsx

```diff
@@ -19,50 +22,67 @@ export const WireDetail = ({ wire }: { wire: WireData }) => {
 			: undefined;
 	}, [wire]);
 
-	const listItems = [
-		{
-			title: 'Byline',
-			description: byline ?? 'Not found',
-		},
-		{
-			title: 'Keywords',
-			description: (
-				<EuiFlexGroup wrap responsive={false} gutterSize="xs">
-					{keywords?.map((keyword) => (
-						<EuiFlexItem key={keyword} grow={false}>
-							<EuiBadge color="primary">{keyword}</EuiBadge>
-						</EuiFlexItem>
-					))}
-				</EuiFlexGroup>
-			),
-		},
-		{
-			title: 'Usage restrictions',
-			description: usage ?? 'Not found',
-		},
-		{
-			title: 'Body text',
-			description: safeBodyText ? (
-				<article dangerouslySetInnerHTML={{ __html: safeBodyText }} />
-			) : (
-				'Not found'
-			),
-		},
-	];
+	const nonEmptyKeywords = useMemo(
+		() => keywords?.filter((keyword) => keyword.trim().length > 0) ?? [],
+		[keywords],
+	);
 
 	return (
 		<Fragment>
-			<h3
-				css={css`
-					font-weight: 300;
-				`}
-			>
-				{wire.content.subhead}
-			</h3>
+			<EuiScreenReaderLive focusRegionOnTextChange>
```

Comment:
this means that the Item box gets focused every time a new item is selected, which feels like the right behaviour for screenreaders and for general keyboard support?